# 現在の運転状態GET
---
uint32 robotStat

